Bio/Seq.py

```diff
@@ -1633,9 +1632,9 @@ def join(self, other):
 
 
 class MutableSeq:
-    """An editable sequence object (with an alphabet).
```

Comment:
Can you put this in a separate PR please, along with the other removal of an alphabet mention?

Reply:
Included on #3192